libs/framework/src/celix_framework_utils.c

```diff
@@ -196,7 +197,23 @@ static celix_status_t celix_framework_utils_extractBundlePath(celix_framework_t
         //other errors should be caught by celix_framework_utils_isBundleUrlValid
         return CELIX_ENOMEM;
     }
-    celix_status_t status = celix_utils_extractZipFile(resolvedPath, extractPath, &err);
+    celix_status_t status = CELIX_SUCCESS;
+    if (celix_utils_directoryExists(resolvedPath)) {
```

Comment:
Nice :) and a shockingly small change to support uncompressed dir (although I know much has happened in other PR to prepare this (e.g. handling of symlinks)).

Reply:
Thanks to the foundational cache refactoring.